handsontable/src/3rdparty/walkontable/src/core/_base.js

```diff
@@ -123,8 +124,10 @@ export default class CoreAbstract {
   draw(fastDraw = false) {
     this.drawInterrupted = false;
 
-    if (!this.wtTable.isVisible()) {
-      // draw interrupted because TABLE is not visible
+    if (!this.wtTable.isVisible() ||
+      (this.wtViewport.getTrimmingContainerHeight() === 0 && hasZeroHeight(this.wtTable.wtRootElement.parentNode))
+    ) {
```

Comment:
I think I would keep that as simple as possible and remove the `getTrimmingContainerHeight` method, as it returns relatively unstable values. For different table configurations, it returns either a number or `undefined` (when there is no top overlay). 
```suggestion
    if (!this.wtTable.isVisible() || hasZeroHeight(this.wtTable.wtRootElement.parentNode)) {
```